parser-service/src/test/java/org/mule/parser/service/ReferencesResolverTestCase.java

```diff
@@ -18,12 +19,13 @@
 import static java.util.Collections.emptyList;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.IsEqual.equalTo;
-import static org.mockito.Matchers.any;
+import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 import static org.mule.apikit.model.api.ApiReference.create;
 
 public class ReferencesResolverTestCase {
 
+  @Ignore
```

Comment:
is there a way to avoid @ignore?

Reply:
great catch Juan,, let's revert this until we can avoid it